ci-operator/config/openshift/csi-driver-nfs/openshift-csi-driver-nfs-master.yaml

```diff
@@ -26,6 +26,13 @@ images:
       as:
       - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
   to: csi-driver-nfs
+- dockerfile_path: images/Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
+  to: nfs-csi-driver
```

Comment:
Wait, I'm confused.  The naming before was preferred - `csi-driver-X`.  Why are we changing it?

Reply:
This is how we decided to call csi drivers. For instance, we have ovirt-csi-driver. This pr renames nfs driver accordingly.

Reply:
@smarterclayton It was done to unify csi driver names in OpenShift and address your concerns in https://github.com/openshift/release/pull/12611